Detectors/FIT/common/calibration/include/FITCalibration/FITCalibrationDevice.h

```diff
@@ -18,14 +18,14 @@
 #include "Framework/WorkflowSpec.h"
 #include "Framework/DataProcessorSpec.h"
 
-namespace o2::calibration::fit
+namespace o2::fit
 {
 
 template <typename InputCalibrationInfoType, typename TimeSlotStorageType, typename CalibrationObjectType>
 class FITCalibrationDevice : public o2::framework::Task
 {
 
-  static constexpr std::size_t OBJECT_SENDING_FREQUENCY = 5;
+  static constexpr std::size_t OBJECT_SENDING_FREQUENCY = 1;
```

Comment:
(Calibration meeting conclusion)
This should be simple as possible, so we do not need this parameter. Simply push object to ccdb-populator if ready.